### ISSUE TITLE
Resetting stats ruins the globe text

### ISSUE THREAD

**Issue by theinterestingstory** [runelite/runelite#7066]
Please check if your issue is not a duplicate by [searching existing issues](https://github.com/runelite/runelite/search?type=Issues)

**Describe the bug**
If I go into the XP tracker tab and reset the XP, the experience globe will stay visible, however, if you hover over it, it will show that the xp needed is negative your current XP.

**To Reproduce**
For example, say I have 12,000,000 current XP and I clean a herb. I reset my xp tracker. If I hover over the globe, it now only says Current XP: 12,000,000 and for XP Left it says: -12,000,000.

**Expected behavior**
I don't know what should be done, but what it is now is a bug. Maybe clear the globe on an XP reset?

**Screenshots**
https://i.imgur.com/47UTnXF.png

**Environment (please complete the following information):**
 - All
 - RuneLite version: 1.5.5
 - Launcher version: 1.6

**Additional context**
n/a

**Comment by JAnken123** [comment]
I can reproduce this, however I think this is not really a big issue? It's not persistent, so gaining new exp in the same skill after reproducing this bug removes this effect and simply waiting also clears the globe as usual.

**Comment by theinterestingstory** [comment]
It's probably one of the less to matter bugs. I just didn't see it listed in here and I thought I should mention it in case it scraped by unnoticed.

**Comment by JAnken123** [comment]
Yes, I had a quick look at the globes code and I think just doing some sort of check and draw same as last time if null in `render` in XpGlobesOverlay would fix it as it seems to update values
```
for (final XpGlobe xpGlobe : sortedXpGlobes)
		{
			int startXp = xpTrackerService.getStartGoalXp(xpGlobe.getSkill());
			int goalXp = xpTrackerService.getEndGoalXp(xpGlobe.getSkill());
			renderProgressCircle(graphics, xpGlobe, startXp, goalXp, curDrawX, 0, getBounds());
			curDrawX += MINIMUM_STEP + config.xpOrbSize();
		}
```
